### ISSUE TITLE
[feature]: Exporting of Pages to different file formats

### ISSUE THREAD

**Issue by jhdcruz** [makeplane/plane#4738]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Allow exporting of Pages to say Markdown or PDFs.

It seems you guys are using Django as backend, perhaps this [library](https://github.com/JessicaTegner/pypandoc) might be able to help.

### Why should this be worked on?

For submitting files to people that doesn't need or not applicable to be invited to the workspace.

Like in our use case, our Professors/Teachers.

**Comment by ejones40** [comment]
Agreed, this would be very valuable. Exporting to PDF would support sharing information to non-plane users, such as a process document.

**Comment by aaryan610** [comment]
Hey folks! 👋
This feature was shipped a while back with #5705. You can now export a page as PDF or Markdown. Give it a try and let us know what you think! 🚀
